lib/ruby_lsp/server.rb

```diff
@@ -0,0 +1,157 @@
+# typed: strict
+# frozen_string_literal: true
+
+require_relative "internal"
```

Comment:
Can we use a regular require here? I like to avoid relative requires as much as possible.

Reply:
I'm fine with both but why did we had `require_relative "handler"` before?